app/models/concept.rb

```diff
@@ -0,0 +1,3 @@
+class Concept < ApplicationRecord
+  belongs_to :question
```

Comment:
Good question, the story wasn't very specific so I just went with how responses on response sets worked. I'm not sure what the UI would look like if you could select previously made concepts. I'll ask for clarification.